convert.lua

```diff
@@ -52,7 +52,7 @@ function get_ip_src()
 	local ipv6_src = ipv6_src_f()
 	if ip_src then 
 		return ip_src
-	elseif ipv6_src then
+	elseif ipv6_src then 
```

Comment:
>  Restore trailing spaces removed by text editor 

I wanted to say: that's good you remove them, they should not be present because many people, like you, have their editor removing these useless trailing space.
But just, next time, please remove them in a dedicated commit, not in one included with other modifications. → So please drop this commit.

Also, always best to use `git add -p` when adding modification to your Git' stage.